pkg/expr/transform.go

```diff
@@ -11,11 +11,35 @@ import (
 	"github.com/grafana/grafana/pkg/components/simplejson"
 	"github.com/grafana/grafana/pkg/models"
 	"github.com/grafana/grafana/pkg/tsdb"
+	"github.com/prometheus/client_golang/prometheus"
 	"golang.org/x/net/context"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 )
 
+var (
+	transformQueryCounter   *prometheus.CounterVec
+	transformQueryHistogram *prometheus.HistogramVec
+)
+
+func init() {
+	transformQueryCounter = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: "grafana",
+		Name:      "transform_queries_total",
+		Help:      "The total number of tranform queries",
+	}, []string{"status"})
+	prometheus.MustRegister(transformQueryCounter)
+
+	transformQueryHistogram = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: "grafana",
+		Name:      "transform_queries_duration_seconds",
+		Help:      "Tranform query histogram",
+		Buckets:   prometheus.ExponentialBuckets(0.0001, 2, 24),
```

Comment:
I'm not completely sure about this bucket configuration; another one could better serve our purposes.

Reply:
Thats alot of buckets :) I think we should reduce the amount. Might make more sense to define these manually and update them if we need more granularity.

Reply:
thanks, I can make configurable the `count`.

Reply:
@bergquist 
I have changed it to:
`Buckets:   prometheus.ExponentialBuckets(0.001, 4, 9),`
do you think that is fine or should we better let the users override it?

Reply:
I'm no export at this. But your suggestion would results in the following buckets ([Go playground](https://play.golang.org/p/dqD5LkgBoXm)): [0.001 0.004 0.016 0.064 0.256 1.024 4.096 16.384 65.536]

This would mean the lowest bucket (0.001) means every transform taking from zero to 0.001ms (1 Microsecond) which seems very low and the highest bucket from 65.536ms and infinity which doesn't seem very useful.

Using for example `ExponentialBuckets(200, 1.5, 9)` would produce the following buckets `[200 300 450 675 1012.5 1518.75 2278.125 3417.1875 5125.78125]` with lowest going from zero to 200ms and highest from ~5s and infinity. 

But at the end it depends on what your normal expected durations are, whether you're planning to use percentiles and/or how you're planning to use the metrics. More information at https://prometheus.io/docs/practices/histograms/#errors-of-quantile-estimation